custom_connectors/custom_auth/kizen.rb

```diff
@@ -56,6 +56,119 @@
+          { key => value }
+        end.inject(:merge)
+      end
+    end,
+
+    format_schema: lambda do |schema|
+      if schema.is_a?(Array)
+        schema.map do |array_value|
+          call('format_schema', array_value)
+        end
+      elsif schema.is_a?(Hash)
+        schema.map do |key, value|
+          if %w[name].include?(key.to_s)
+            value = call('replace_special_characters', value.to_s)
+          elsif %w[properties toggle_field].include?(key.to_s)
+            value = call('format_schema', value)
+          end
+          { key => value }
+        end.inject(:merge)
+      end
+    end,
+
+    replace_special_characters: lambda do |input|
+      input.gsub(/[-<>!@#$%^&*()+={}:;'"`~,.?|]/,
+                 '-' => '__hyp__',
+                 '<' => '__lt__',
+                 '>' => '__gt__',
+                 '!' => '__excl__',
+                 '@' => '__at__',
+                 '#' => '__hashtag__',
+                 '$' => '__dollar__',
+                 '%' => '__percent__',
+                 '^' => '__pwr__',
+                 '&' => '__amper__',
+                 '*' => '__star__',
+                 '(' => '__lbracket__',
+                 ')' => '__rbracket__',
+                 '+' => '__plus__',
+                 '=' => '__eq__',
+                 '{' => '__rcrbrack__',
+                 '}' => '__lcrbrack__',
+                 ';' => '__semicol__',
+                 '\'' => '__apost__',
+                 '`' => '__bckquot__',
+                 '~' => '__tilde__',
+                 ',' => '__comma__',
+                 '.' => '__period__',
+                 '?' => '__qmark__',
+                 '|' => '__pipe__',
+                 ':' => '__colon__',
+                 '\"' => '__quote__')
+    end,
+
+    inject_special_characters: lambda do |input|
+      input.gsub(
+        /(__hyp__|__lt__|__gt__|__excl__|__at__|__hashtag__|__dollar__|\__percent__|__pwr__|__amper__|__star__|__lbracket__|__rbracket__|__plus__|__eq__|__rcrbrack__|__lcrbrack__|__semicol__|__apost__|__bckquot__|__tilde__|__comma__|__period__|__qmark__|__pipe__|__colon__|__quote__|__slash__|__bslash__)/,
+        '__hyp__' => '-',
+        '__lt__' => '<',
+        '__gt__' => '>',
+        '__excl__' => '!',
+        '__at__' => '@',
+        '__hashtag__' => '#',
+        '__dollar__' => '$',
+        '__percent__' => '%',
+        '__pwr__' => '^',
+        '__amper__' => '&',
+        '__star__' => '*',
+        '__lbracket__' => '(',
+        '__rbracket__' => ')',
+        '__plus__' => '+',
+        '__eq__' => '=',
+        '__rcrbrack__' => '{',
+        '__lcrbrack__' => '}',
+        '__semicol__' => ';',
+        '__apost__' => '\'',
+        '__bckquot__' => '`',
+        '__tilde__' => '~',
+        '__comma__' => ',',
+        '__period__' => '.',
+        '__qmark__' => '?',
+        '__pipe__' => '|',
+        '__colon__' => ':',
+        '__quote__' => '"'
+      )
+    end,
+
+    format_response: lambda do |payload|
```

Comment:
This has been replaced with the `format_output` method. Could we use `format_output` in the new actions and triggers?

Reply:
Deleted format_response and changed references to format_output